### ISSUE TITLE
Demo test have label but do not have boxes

### ISSUE THREAD

**Issue by lizituo12** [facebookresearch/SlowFast#280]
the code is this:
['cfg','../configs/Kinetics/SLOWFAST_8x8_R50.yaml','DETECTION.ENABLE','True','TRAIN.ENABLE','False','DEMO.ENABLE','True','NUM_GPUS','1']
and the yaml file like ths:
TRAIN:
ENABLE: False
DATASET: kinetics
BATCH_SIZE: 64
EVAL_PERIOD: 10
CHECKPOINT_PERIOD: 1
CHECKPOINT_FILE_PATH: "../checkpoint/Kinetic/SLOWFAST_8x8_R50.pkl"
CHECKPOINT_TYPE: caffe2
AUTO_RESUME: True

DATA:
NUM_FRAMES: 32
SAMPLING_RATE: 2
TRAIN_JITTER_SCALES: [256, 320]
TRAIN_CROP_SIZE: 224
TEST_CROP_SIZE: 256
INPUT_CHANNEL_NUM: [3, 3]
SLOWFAST:
ALPHA: 4
BETA_INV: 8
FUSION_CONV_CHANNEL_RATIO: 2
FUSION_KERNEL_SZ: 7
RESNET:
ZERO_INIT_FINAL_BN: True
WIDTH_PER_GROUP: 64
NUM_GROUPS: 1
DEPTH: 50
TRANS_FUNC: bottleneck_transform
STRIDE_1X1: False
NUM_BLOCK_TEMP_KERNEL: [[3, 3], [4, 4], [6, 6], [3, 3]]
SPATIAL_STRIDES: [[1, 1], [2, 2], [2, 2], [2, 2]]
SPATIAL_DILATIONS: [[1, 1], [1, 1], [1, 1], [1, 1]]
NONLOCAL:
LOCATION: [[[], []], [[], []], [[], []], [[], []]]
GROUP: [[1, 1], [1, 1], [1, 1], [1, 1]]
INSTANTIATION: dot_product
BN:
USE_PRECISE_STATS: True
NUM_BATCHES_PRECISE: 200
SOLVER:
BASE_LR: 0.1
LR_POLICY: cosine
MAX_EPOCH: 196
MOMENTUM: 0.9
WEIGHT_DECAY: 1e-4
WARMUP_EPOCHS: 34.0
WARMUP_START_LR: 0.01
OPTIMIZING_METHOD: sgd
MODEL:
NUM_CLASSES: 400
ARCH: slowfast
MODEL_NAME: SlowFast
LOSS_FUNC: cross_entropy
DROPOUT_RATE: 0.5
TEST:
ENABLE: False
DATASET: kinetics
BATCH_SIZE: 64
DATA_LOADER:
NUM_WORKERS: 8
PIN_MEMORY: True
DEMO:
ENABLE: True
LABEL_FILE_PATH: "../class.json"
INPUT_VIDEO: "E:/project/dataset/action_recognition/UCF_crime_durings/Normal_Videos_event/Normal_Videos_event/Normal_Videos052_x264.mp4"
OUTPUT_FILE: "./normal_output.mp4"

DISPLAY_WIDTH: 640
DISPLAY_HEIGHT: 480
WEBCAM: 0
DETECTRON2_CFG: "e:/project/detection2/configs/COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"
DETECTRON2_WEIGHTS: "../checkpoint/DETECTION2/R-50.pkl"
NUM_GPUS: 0
NUM_SHARDS: 1
RNG_SEED: 0
OUTPUT_DIR: .

the code run well but the result do not draw the boxes out ;
i think it may because i do not scaled the data or the threshold but where should i to change it ,thanks

image

**Comment by nikky4D** [comment]
I have a similar question. Do I plot the detectron2 results separately or is that incorporated into pyslowfast and if so, how do I plot the bounding boxes/regions?

**Comment by lizituo12** [comment]
> I have a similar question. Do I plot the detectron2 results separately or is that incorporated into pyslowfast and if so, how do I plot the bounding boxes/regions?

I have fixed this problem, just set it in your configuration file

DETECTION:

ENABLE: True

ALIGNED: False

And set the

DETECTRON2_CFG: "COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"

DETECTRON2_WEIGHTS: ".. /checkpoint/DETECTRON2/model_final_280758.pkl"

Just fine.

**Comment by wwdok** [comment]
> > I have a similar question. Do I plot the detectron2 results separately or is that incorporated into pyslowfast and if so, how do I plot the bounding boxes/regions?
> 
> I have fixed this problem, just set it in your configuration file
> 
> DETECTION:
> 
> ENABLE: True
> 
> ALIGNED: False
> 
> And set the
> 
> DETECTRON2_CFG: "COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"
> 
> DETECTRON2_WEIGHTS: ".. /checkpoint/DETECTRON2/model_final_280758.pkl"
> 
> Just fine.

Could you please share the download link of `model_final_280758.pkl` ? thanks ~

**Comment by wwdok** [comment]
Hi, could you please share you class.json file ? The official link file doesn't not exist anymore
